dev/src/sample.test.html

```diff
@@ -1,432 +1,560 @@
 <!DOCTYPE html>
-<html lang="en" dir="ltr">
+<html lang="en" dir="ltr" class="su-smooth-scroll">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Decanter Preview Environment</title>
-</head>
+  <link href="/decanter.css" rel="stylesheet"><link href="/test.css" rel="stylesheet"></head>
```

Comment:
You don't need to add these. Webpack does it for you.
https://github.com/SU-SWS/decanter/blob/v7/dev/webpack.dev.config.js#L85